datahub-web-react/src/app/shared/tags/EditDescriptionModal.tsx

```diff
@@ -0,0 +1,71 @@
+import { Typography, Modal, Button, Form } from 'antd';
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import MDEditor from '@uiw/react-md-editor';
+
+const MarkDownHelpLink = styled(Typography.Link)`
+    position: absolute;
+    right: 0;
+    top: -18px;
+    font-size: 12px;
+`;
+
+type Props = {
+    title: string;
+    description?: string | undefined;
+    onClose: () => void;
+    onSubmit: (description: string) => void;
+    isAddDesc?: boolean;
+};
+
+export default function EditDescriptionModal({ title, description, onClose, onSubmit, isAddDesc }: Props) {
+    const [updatedDescription, setUpdatedDescription] = useState(description || '');
+
+    return (
+        <Modal
+            title={title}
+            visible
+            width={900}
+            onCancel={onClose}
+            okText={isAddDesc ? 'Submit' : 'Update'}
```

Comment:
there is already the `src/app/entity/shared/components/legacy/DescriptionModal.tsx` Can we use that instead? if not, can we refactor it so it can be used by both schemas and tags?

Reply:
Fixed and commited.

[https://github.com/linkedin/datahub/pull/4132/commits/efdfaedb45210bec45c43f2d0f00f5f919c4a62a](url)